docs/blog/plots-in-tables/index.qmd

```diff
@@ -0,0 +1,242 @@
+---
+title: "Adding Plots to Great Tables"
+html-table-processing: none
+author: Jules Walzer-Goldfeld and Michael Chow
+date: 2025-07-03
```

Comment:
do you mind adding `freeze: true` to the topmatter? Then, if you run quarto render, you should be able to add a folder called something like `_freeze/blog/plots-in-tables`.